modules/installation-configuration-parameters.adoc

```diff
@@ -1558,6 +1558,11 @@ Additional {ibmpowerProductName} Virtual Server configuration parameters are des
 1. Whether you define an existing resource group, or if the installer creates one, determines how the resource group is treated when the cluster is uninstalled. If you define a resource group, the installer removes all of the installer-provisioned resources, but leaves the resource group alone; if a resource group is created as part of the installation, the installer removes all of the installer provisioned resources and the resource group.
 2. To determine which profile best meets your needs, see https://cloud.ibm.com/docs/vpc?topic=vpc-profiles&interface=ui[Instance Profiles] in the IBM documentation.
 --
+
+[NOTE]
+====
+Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region as they have been replaced by the `Power Edge Router`.
```

Comment:
```suggestion
Cloud connections are no longer supported in the `install-config.yaml` while deploying in the `dal10` region, as they have been replaced by the `Power Edge Router`.
```

Reply:
Making sure Power Edge Router is [supposed to be in backticks](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) (is an API object or a system item). I'm not sure what it is, so I thought I'd ask!

Reply:
@kcarmichael08 okay. I have removed the backticks. Thanks.

FYI: A Power Edge Router (PER) is a high-performance router that provides advanced routing capabilities for IBM® Power® Systems Virtual Server users. Reference: https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-per